bibliopixel/drivers/visualizer.py

```diff
@@ -1,11 +1,6 @@
-import time
-
-import os
-import platform
-import subprocess
-from network import DriverNetwork, socket
-import math
+import time, os, platform, subprocess, math
```

Comment:
Side note. Moving forward I'm planning on having everything be PEP8 compliant to make my linter happy (minus a few rules I disagree with - E501, E401 mainly). So each import on it's own line.